pkg/apiserver/bootstrap.go

```diff
@@ -132,13 +132,17 @@ func bootstrap(params APIServerParams) error {
 
 	// if no status informers are provided, generate them from the helm release
 	informers := params.StatusInformers
-	if len(informers) == 0 && helm.IsHelmManaged() {
-		helmRelease, err := helm.GetRelease(helm.GetReleaseName())
-		if err != nil {
-			return errors.Wrap(err, "failed to get helm release")
-		}
-		if helmRelease != nil {
-			informers = appstate.GenerateStatusInformersForManifest(helmRelease.Manifest)
+	if informers == nil {
+		informers = []appstatetypes.StatusInformerString{}
+		if helm.IsHelmManaged() {
```

Comment:
modified some logic slightly here specifically for the `helm template` + `kubectl apply` use-case. in that case, `nil` would have been passed to `ApplyAppInformers`, which behaved as expected, but did not feel right

Reply:
it's fine if `ApplyAppInformers` can handle nil status informers. up to you, but i liked the old structure better :)